staging/src/k8s.io/kube-gen/test/apis/testgroup/v1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,130 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Did you rebase to master?  kube-gen was renamed last week.

git fetch upstream
git rebase upstream/master

All of your changes related to kube-gen should be removed, including the lint_failures